Dockerfile

```diff
@@ -59,6 +59,6 @@ COPY --from=build --chown=nobody:nobody /app/_build/prod/rel/membrane_dashboard
 
 ENV HOME=/app
 
-EXPOSE 4000
+EXPOSE 3456
```

Comment:
Can we have more popular port like `8000`? I know that we change `4000` as it is a default for phoenix applications. If somebody wants to change the port while using docker he won't have any problems and when running locally then you just have to specify it yourself.

Reply:
okay, we can make it 8000